functional-tests/fn_prover_el_dispatch.py

```diff
@@ -18,7 +18,7 @@ def main(self, ctx: flexitest.RunContext):
         time.sleep(60)
 
         # Dispatch the prover task
-        task_id = prover_client_rpc.dev_strata_proveELBlock(1)
+        task_id = prover_client_rpc.dev_strata_proveElBlocks((1, 3))
```

Comment:
Why the magic numbers?
Can you comment on them in the file for the poor soul who will have to edit/maintain this for posterity?